pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -409,9 +414,12 @@ func (s *Service) Name() string {
 }
 
 // Connection return connection
-func (s *Service) Connection(id string) {
-	// TODO add Connection logic
-
+func (s *Service) Connection(connectionID string) (*ConnectionRecord, error) {
```

Comment:
I wonder if we should prefix this method to indicate that this is a query/lookup rather than just an accessor?

Reply:
I will rename it to QueryConnectionByID like the client

Reply:
done